src/ResourcesGenerator/ResourceCreator.cs

```diff
@@ -51,6 +51,7 @@ public class ResourceCreator
             "TaskFoundFromFactory",
             "TaskFound",
             "PropertyReassignment",

```

Comment:
@KirillOsenkov could you please tell me why do you have duplicates in this array?
e.g. "PropertyReassignment"

Reply:
that's just an accident, we can remove